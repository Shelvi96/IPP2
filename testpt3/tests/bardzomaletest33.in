NEW ID

7>8560539
:047>505136
82818060>508
687>680:4;
1322>063629
23638>;
4056:>:8299303
;3479:4>1;28;0;95;
3>3195
:1213>871
591:836776>0967;98
72;>94;
@:82753:5:806
@0:4:6776769:2635187
@:7665;813607545;
@49:;69144;906:2540
@394:34500:;339127
@746:1690128
@14252:18:1:241
@57:872658120658;60
@374
@329541;7:66:10570:90861701170
@37801450086:602373
@8393497:
@7:4:6
@3;88;089:39
@68409;14
