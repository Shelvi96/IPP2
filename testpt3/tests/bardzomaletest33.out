0
30636
1020
59778
10110
0
0
60507
0
188845365549514752
2001
0
0
0
0
